driver-core/src/main/com/mongodb/internal/operation/AsyncChangeStreamBatchCursor.java

```diff
@@ -227,23 +226,29 @@ public void onResult(final List<RawBsonDocument> rawDocuments, final Throwable t
         void apply(AsyncAggregateResponseBatchCursor<RawBsonDocument> cursor, SingleResultCallback<List<RawBsonDocument>> callback);
     }
 
-    private void resumeableOperation(final AsyncBlock asyncBlock, final SingleResultCallback<List<RawBsonDocument>> callback,
+    private void resumeableOperation(final AsyncBlock asyncBlock, final SingleResultCallback<List<RawBsonDocument>> convertResultsCallback,
```

Comment:
Renamed `callback` to `convertResultsCallback` to make it more clear what the callback does. While one may guess this by staring at the original callback type `SingleResultCallback<List<RawBsonDocument>>` and the callback type `SingleResultCallback<List<T>>`, the variable name in this case does help because the converting callback type does not have a custom name.